galasa-parent/dev.galasa.framework.api.ras/openapi.yaml

```diff
@@ -276,6 +276,8 @@ components:
             type: string
     TestStructure:
       properties:
+        runId:
```

Comment:
run only,   should not be in the test structure,  as the runid is specific to the RAS,   not the actual test itself